hardware_layer/hardware_arduino/scripts/serial_node.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
File not required. Use rosnode directly. See fourtran.

Reply:
Removed the file and made as per fourtran